benchmarks/benchmarks/bench_shape_base.py

```diff
@@ -65,6 +65,24 @@ def time_no_lists(self, n):
         np.block(np.eye(3 * n))
 
 
+class Block2D(Benchmark):
+    params = [[(16, 16), (32, 32), (64, 64), (128, 128), (256, 256), (512, 512), (1024, 1024)],
+              ['uint8', 'uint16', 'uint32', 'uint64'],
+              [(2, 2), (4, 4)]]
+    param_names = ['shape', 'dtype', 'n_chunks']
+
+    def setup(self, shape, dtype, n_chunks):
+
+        self.block_list = [
+             [np.full(shape=[s//n_chunk for s, n_chunk in zip(shape, n_chunks)],
```

Comment:
Why `full` and not just `ones`?

Reply:
Honestly, I think it is more obvious what we are trying to do with `full` since it implies array traversal in the name. `ones`, `zeros` and `emtpy` are ambiguous since they do different things.